{
    "identifier": "CVE-2017-1000433",
    "summary": "pysaml2 version 4.4.0 and older accept any password when run with python optimizations enabled. This allows attackers to log in as any user without knowing their password.",
    "references": [
        {
            "url": "",
            "reference_id": "pyup.io-35700"
        }
    ],
    "vulnerable_packages": [
        "pkg:pypi/pysaml2@4.2.0",
        "pkg:pypi/pysaml2@0.4.3",
        "pkg:pypi/pysaml2@4.4.0",
        "pkg:pypi/pysaml2@2.1.0",
        "pkg:pypi/pysaml2@2.0.0",
        "pkg:pypi/pysaml2@3.0.2",
        "pkg:pypi/pysaml2@4.0.2",
        "pkg:pypi/pysaml2@4.0.3",
        "pkg:pypi/pysaml2@1.1.0",
        "pkg:pypi/pysaml2@1.0.1",
        "pkg:pypi/pysaml2@2.3.0",
        "pkg:pypi/pysaml2@4.0.0",
        "pkg:pypi/pysaml2@2.4.0",
        "pkg:pypi/pysaml2@4.1.0",
        "pkg:pypi/pysaml2@4.3.0",
        "pkg:pypi/pysaml2@2.2.0",
        "pkg:pypi/pysaml2@4.0.5rc1",
        "pkg:pypi/pysaml2@3.0.0",
        "pkg:pypi/pysaml2@4.0.5",
        "pkg:pypi/pysaml2@1.0.3",
        "pkg:pypi/pysaml2@1.0.2",
        "pkg:pypi/pysaml2@4.0.4",
        "pkg:pypi/pysaml2@4.0.1"
    ],
    "resolved_packages": [
        "pkg:pypi/pysaml2@5.0.0",
        "pkg:pypi/pysaml2@4.6.0",
        "pkg:pypi/pysaml2@4.6.4",
        "pkg:pypi/pysaml2@5.3.0",
        "pkg:pypi/pysaml2@4.8.0",
        "pkg:pypi/pysaml2@6.1.0",
        "pkg:pypi/pysaml2@4.7.0",
        "pkg:pypi/pysaml2@4.6.3",
        "pkg:pypi/pysaml2@4.6.5",
        "pkg:pypi/pysaml2@4.6.1",
        "pkg:pypi/pysaml2@5.4.0",
        "pkg:pypi/pysaml2@4.9.0",
        "pkg:pypi/pysaml2@6.0.0",
        "pkg:pypi/pysaml2@5.2.0",
        "pkg:pypi/pysaml2@5.1.0",
        "pkg:pypi/pysaml2@4.6.2",
        "pkg:pypi/pysaml2@4.5.0"
    ]
}